config/routes.rb

```diff
@@ -1,8 +1,12 @@
 Rails.application.routes.draw do
   get 'users/index'
+  
+  get 'users/show'
```

Comment:
your user defined `user` routes should be below the devise_for :users route to avoid conflicts